.luacheckrc

```diff
@@ -1,6 +1,16 @@
-std = 'ngx_lua+lua52' -- lua52 has table.pack
+local standards = require "luacheck.standards"
+
+stds.ngx = {
```

Comment:
Where does this `stds` table come from?

Reply:
Provided by luacheck.

https://luacheck.readthedocs.io/en/stable/config.html#custom-sets-of-globals

Should I link it in the file?

Reply:
I think we should